pkg/errors/errors.go

```diff
@@ -94,3 +102,14 @@ func New(text string) Error {
 		err: nil,
 	}
 }
+
+func KillSignalHandler(ctx context.Context) error {
```

Comment:
Consider renaming to `SignalHandler` since this actually does not handle kill signal. KILL signal cannot be trapped. Maybe we should also add `SIGABRT` here, as well.